multiwallet_utils.go

```diff
@@ -150,7 +150,8 @@ func decryptWalletSeed(pass []byte, encryptedSeed []byte) (string, error) {
 
 	decryptedSeed, err := secretbox.EasyOpen(encryptedSeed, key)
 	if err != nil {
-		return "", err
+		log.Error(err)
```

Comment:
Maybe add more info to the logged error to make it searchable and more informative, something like:
```suggestion
		log.Errorf("decryptWalletSeed secretbox.EasyOpen error: %v", err)
```